codeexamples/SYCLUsingCUDAStaticLibraryOfKernels/README.md

```diff
@@ -0,0 +1,39 @@
+# A SYCL program executing CUDA kernels
+
+This project shows how to call a CUDA __global__ kernel from within a SYCL program using one task graph. The global kernel makes further to kernel calls to CUDA __device__ functions. The program is version of the original CUDA sample **dxtc** found in the NVIDIA CUDA samples github repository [here](https://github.com/NVIDIA/cuda-samples/tree/master/Samples/5_Domain_Specific/dxtc).
+
+DXTC stands for High Quality DirectX Compressor. This example uses the same framework (a SYCL framework) and CUDA kernel functions as the original.
+
+This SYCL example is based on the SYCL USM framework model. Refer to the Codeplay CUDA interopability with SYCL example **SYCLBufferUsingStaticLibraryCUDAkernels** to see how a SYCL buffer framework model is used. Unlike the **SYCLBufferUsingStaticLibraryCUDAkernels** example this example does not mix SYCL kernel execution with CUDA kernels in the same task graph (the one queue) to act on one set of data to create some new data or results.
+
+## CUDA interopability approach with SYCL
+All the CUDA kernels are put in a static library. The global kernel is wrapped by a C++ Facade function. This library is compiled with the NVIDIA compiler.
+
+The static library is linked to a SYCL project. The Facade wrapper function is called from within a SYCL host_task() fucntion, which in turn is enqueued in a SYCL queue.
+
+## Microsoft's Visual Studio Code IDE (VSCode)
+VSCode orgranises each and every individual C++ project with a set of settings and configuration files. These files are contained in a hidden directory named **.vscode**. The configuration files **tasks.json** and **launch.json** are used to configure the building (generally compiling) of the project  followed by the debugging the program respectively. The other files like **settings.json** and **c_cpp_properties.json** are used by VSCode to inform it of your preferences and development environment. To learn more about how to use VSCode with DPC++, see the series of VSCode blogs at [here](https://codeplay.com/portal/blogs/2023/03/01/setting-up-c-development-with-visual-studio-code-on-ubuntu).
+
+### VSCode and git
+As VSCode configuration files may have settings that are local to your development environment, you are likely to want to keep them, but not likely want them to be staged to be commited and so pushed up stream, nor have git overwrite your local settings.
+For the case of not wanting to push local changes, the git .gitignore file handles this. For case where local settings are to be kept you may want to do the following: 
+Perform a git stash, before getting the latest changes from the repository (by using git pull origin master or git rebase origin/master), and then merge your changes from the stash using git stash pop stash@{0}.
```

Comment:
I'm not sure we need these instructions here (I don't think we're likely to take many pull requests from external collaborators).

Reply:
Where would you like these instructions to be positioned in the repository if not here? ATM there will only a couple of examples. These instructions cannot be isolated from the context they are trying to inform the user.